frontend/templates/views/partials/consent.j2

```diff
@@ -2,7 +2,8 @@
   <script type="application/json">
     {
       "ISOCountryGroups": {
-        "eu": ["at", "be", "bg", "ch", "cy", "cz", "dk", "de", "ee", "el", "es", "fi", "fr", "hr", "ie", "is", "it", "li", "lv", "lt", "lu", "hu", "mt", "nl", "no", "pl", "pt", "ro", "si", "sk", "se", "uk"]
+        "eu": ["at", "be", "bg", "ch", "cy", "cz", "dk", "de", "ee", "el", "es", "fi", "fr", "hr", "ie", "is", "it", "li", "lv", "lt", "lu", "hu", "mt", "nl", "no", "pl", "pt", "ro", "si", "sk", "se", "uk", "gr"],
```

Comment:
Greece has been missing here, are there reasons for it? Initially I've copied this from the ampproject.org implementation.

Reply:
It's best to use the [eea preset](https://amp.dev/documentation/components/amp-geo?referrer=ampproject.org#preset-country-groups)

Reply:
Sounds good, wasn't to sure if there were reasons for the individually maintained list.